### PR TITLE
PMM-7723 update grafana dashboards branching and guideline

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,14 +1,44 @@
...
+
+=== 1. Sign the CLA
+
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/percona/grafana-dashboards) (CLA). You can do this using your GitHub account and one click.
+
+=== 2. Code of Conduct
+
+Please make sure to read and agree to our [Code of Conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
+
+== Submitting a Bug
```
Comment (denisok) [inline CONTRIBUTING.md:20]: we have issues enabled in this project as well, so maybe make jira optional.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,44 +74,52 @@ docker-compose -f ./docker-compose.local.yml up -d
 npm run dev
 ```
 
-## Workflow for the contributor
+== Submitting a Pull Request
 
 1.  Find the task in [JIRA](https://jira.percona.com/issues/?jql=project+%3D+PMM+AND+component+%3D+%22Grafana+Dashboards%22) or create a new task. Use a component filter Grafana Dashboards.
```
Comment (denisok) [inline CONTRIBUTING.md:79]: or issue here in GH

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -44,44 +74,52 @@ docker-compose -f ./docker-compose.local.yml up -d
...
 
 2.  You need to make a fork of our repository in your GitHub account.
 
 3.  Make a clone of your repository on your computer.
 
-4.  Switch the repository to the correct branch. Now it's PMM-2.0.
+4.  Create a new branch for your task. Name it using the following pattern: [JIRA_ISSUE_ID]-[username]-[short_title]. For example: PMM-5053-dbazhenov-tooltip .
```
Comment (denisok) [inline CONTRIBUTING.md:85]: we can relax branch naming, it is up to ppl how to call hem, we could recommend as best practice to mention issue number or just short desc

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -44,44 +74,52 @@ docker-compose -f ./docker-compose.local.yml up -d
...
+12. Our Engineers will merge your branch into the release branch by themselves.
+
+== After your Pull Request is merged
+
+Once your pull request is merged, you are an official Percona Community Contributor. Welcome to the community!
 
-13. Our Engineers will merge your branch into the release branch by themselves.
 
-14. Congratulations, you have become a contributor. Thanks for contributing to open source!
+We're looking forward to your contributions and hope to hear from you soon on our [Forums](https://forums.percona.com) and [Discord](https://discord.gg/mQEyGPkNbR).
```
Comment (denisok) [inline CONTRIBUTING.md:125]: same for discord link

**Review by denisok — CHANGES_REQUESTED** [review]
nice changes!

**Comment by denisok** [comment]
adding Paul to check lang, I seen couple of issues but he is better for that

**Review by PaulJacobs-percona — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Pull request merged**